dataplane/linux/int_dataplane.go

```diff
@@ -456,6 +461,26 @@ type ifaceUpdate struct {
 	State ifacemonitor.State
 }
 
+// Check if current felix ipvs config is correct when felix gets an kube-ipvs0 interface update.
+// If KubeIPVSInterface is UP and felix ipvs support is disabled (kube-proxy switched from iptables to ipvs mode),
+// or if KubeIPVSInterface is DOWN and felix ipvs support is enabled (kube-proxy switched from ipvs to iptables mode),
+// restart felix to pick up correct ipvs support mode.
+func (d *InternalDataplane) checkIPVSConfigOnIfaceUpdate(update *ifaceUpdate) {
+	if update.Name != KubeIPVSInterface {
```

Comment:
You do this check outside the function too, might as well only do it in one place?

Reply:
Yes. Will do that.